docs/styles.css

```diff
@@ -4,850 +4,458 @@ Fonts
+.calibre-thin {
+  font-family: "Calibre-Thin", sans-serif;
 }
 
-.code-lab.codelab-category-beginner a {
-    border-left: 4px solid #02e68f;
+.pre-wrap {
+  white-space: pre-wrap;
 }
 
-.code-lab.codelab-category-intermediate a {
-    border-left: 4px solid #fbe752;
+.sticky {
+  position: sticky;
 }
 
-.code-lab.codelab-category-advanced a {
-    border-left: 4px solid #fc1452;
+.before-rw-arrow::before {
+  content: "\2192";
 }
 
-.code-lab.codelab-category-beginner h2 {
-    color: #02e68f;
+.after-rw-arrow::after {
+  content: "\2192";
 }
 
-.code-lab.codelab-category-intermediate h2 {
-    color: #fbe752;
+.before-dw-arrow::before {
+  content: "\2193";
 }
 
-.code-lab.codelab-category-advanced h2 {
-    color: #fc1452;
+.hover-before-white:hover::before {
+  color: var(--white);
 }
 
-/* Error page */
-.terminal-title {
-    height: 2em;
-    background-color: #303337;
-    width: 100%;
-    margin: 0;
-    color: white;
-    font-weight: bold;
-    text-align: center;
-    border-radius: 5px 5px 0 0;
-    line-height: 2em;
-}
-.error {
-    width: 800px;
-    margin: 15em auto;
-    position: relative;
-    border-radius: 5px;
-    font-size: 1.2em;
+.b--jet-gray {
+  border-color: var(--jet-gray);
 }
 
-.error .window-buttons {
-    margin-top: 0.5em;
-    float: right;
+.b--turquoise {
+  border-color: var(--turquoise);
 }
+/* END */
 
-.error .window-buttons > a {
-    width: 0.7em;
-    height: 0.7em;
-    border-radius: 1em;
-    float: right;
-    margin-left: 0.2em;
-    margin-right: 0.2em;
-    margin-top: 0.15em;
+/**
+ * BLOCK-LIKE CLASSES
+ */
+/* Code highlighting */
+.build-rule {
+  color: var(--green);
 }
-
-.red-button {
-    background-color: red;
+.fn-name {
+  color: var(--blue);
 }
-
-.green-button {
-    background-color: green;
+.fn-p {
+  color: var(--yellow);
 }
-
-.yellow-button {
-    background-color: yellow;
+.fn-arg {
+  color: var(--green);
 }
-
-.build-stopped {
-    background-color: red;
-    color: white;
-    font-weight: bold;
+.fn-str {
+  color: var(--violet);
 }
-.failed-target {
-    color: red;
-    font-weight: bold;
+.grammar-string {
+  color: var(--yellow);
 }
-
-.error pre {
-    margin: 0;
-    border-radius: 0 0 5px 5px;
-    border: 2px solid #303337;
-    border-top: none;
-    white-space: pre-wrap;
+.grammar-syntax {
+  color: var(--pink);
 }
-
-.error pre a {
-    color: #29abe2;
+.grammar-comment {
+  color: var(--silver);
 }
-
-.cursor {
-    animation: blinker 1s linear infinite;
+.grammar-token {
+  color: var(--green);
+}
+
+/* Codelab */
+.codelab-category-beginner {
+  border-color: var(--turquoise);
+}
+.codelab-category-beginner h2 {
+  color: var(--turquoise);
+}
+.codelab-category-intermediate {
+  border-color: var(--yellow);
+}
+.codelab-category-intermediate h2 {
+  color: var(--yellow);
+}
+.codelab-category-advanced {
+  border-color: var(--red);
+}
+.codelab-category-advanced h2 {
+  color: var(--red);
+}
+
+/* Skip to content */
+.skip-to-content {
```

Comment:
Are these used anywhere?

Reply:
Yes. Load the page and press tab twice (the first focusable element is the logo). It shows a `Skip to content` block that once pressed takes you to the main content. This is for accessibility matters, so the user doesn't have to navigate through the whole menu everytime.